assets/src/domain/shared/services/publicPath.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable 
+no-unused-vars,
+no-native-reassign,
+no-global-assign,
+@typescript-eslint/camelcase,
+@typescript-eslint/no-unused-vars,
+no-var
+*/
```

Comment:
these are similar to magic comments?
https://webpack.js.org/api/module-methods/#magic-comments

Reply:
ESLint and TS don't like that global variable being undefined and modified.